reporting/plex_netflix_check.py

```diff
@@ -29,15 +32,14 @@
 from xmljson import badgerfish as bf
 from lxml.html import fromstring
 from time import sleep
-import json
```

Comment:
This didn't appear to be used, either from the tools or manually looking over the code.